packages/web-client/app/components/card-pay/safe-chooser-dropdown/safe-option/index.hbs

```diff
@@ -0,0 +1,20 @@
+<div class='safe-option' ...attributes>
+  {{#if this.data.icon}}
+    {{svg-jar this.data.icon width="20" height="20" class='safe-option__logo'}}
+  {{else}}
+    <CardPay::MerchantLogo
+      class='safe-option__logo'
+      @name={{first-char this.data.info.name}}
+      @marginRight={{"0"}}
+      @logoBackground={{this.data.info.backgroundColor}}
+      @logoTextColor={{this.data.info.textColor}}
```

Comment:
I think these would probably be cleaner as just css vars rather than a mix of component props and css vars

Reply:
`@logoBackground` and `@logoTextColor` were already there but I’ve changed to using a CSS variable for the right margin, thanks.